doc/sphinx/how-to/how-to-test-mir-for-a-release.md

```diff
@@ -108,6 +108,44 @@ For each empty box in the matrix above, ensure that the following applications c
    gnome-terminal
    ```
 
+5. Test that `X11` apps can be started and can receive input when `XWayland` is supported:
```

Comment:
I think using a specific compositor is appropriate here: e.g. the default `miral-app`